packages/pci-sdk-iframe/src/pages/app/useApp.ts

```diff
@@ -40,20 +40,22 @@ export default function useApp() {
 
 	useEffect(() => {
 		function onMessage(event: MessageEvent) {
-			const data = JSON.parse(event.data);
-			switch (data.type) {
-				case 'setStyle':
-					const style = themeService.extendTheme(data.style);
-					return setState(s => ({ ...s, theme: style }));
-				case 'setTheme':
-					return setState(s => ({ ...s, theme: themes[data.theme as IThemeName] }));
-				case 'showCardData':
-					return showCardData(state.cardId);
-				case 'hideCardData':
-					return setState(s => ({ ...s, networkStatus: 'SUCCESS', pan: `•••• •••• •••• ${s.lastFour}`, cvv: '•••', exp: '••/••' }))
-				default:
-					break;
-			}
+			try {
```

Comment:
This needs to be fixed. In the tests `window.parent.postMessage('apto-iframe-ready', '*')` is caught by `onMessage` and, since it isn't JSON, JSON.parse throws an error.